package.json

```diff
@@ -28,6 +28,7 @@
     "babel-core": "^5.2.17",
     "babel-eslint": "^4.0.5",
     "babelify": "^6.0.0",
+    "backbone.marionette": "^2.4.4",
```

Comment:
It is dev-dependency. It will be installed only for development and tests, like Babel and other modules above.